.github/workflows/pr-container-build.yaml

```diff
@@ -46,7 +49,9 @@ jobs:
           echo "✓"
 
   pr-build:
-    name: PR Build
+    name: PR Publish
+    # if commit message has [ci skip] or [skip ci] don't build + publish this PR
+    if: ${{ ! contains(github.event.head_commit.message, '[ci skip]') }} && ${{ ! contains(github.event.head_commit.message, '[skip ci]') }}
```

Comment:
If commit message has `[ci skip]` or `[skip ci]`, the whole Workflow will be skipped. See https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

Reply:
Ah, I just noticed that this won't stop a workflow that's triggered `on: pull_request_target` from running..

Reply:
yeah :( that's annoying.